ecc/bls12-377/hash_to_g1.go

```diff
@@ -175,38 +87,38 @@ func MapToCurve1(u *fp.Element) G1Affine {
 	var tv3 fp.Element
 	var tv4 fp.Element
 	tv4.SetOne()
-	tv3.Add(&tv2, &tv4)                // 5.  tv3 = tv2 + 1
-	tv3.Mul(&tv3, &sswuIsoCurveCoeffB) // 6.  tv3 = B * tv3
+	tv3.Add(&tv2, &tv4)               // 5.  tv3 = tv2 + 1
+	tv3.Mul(&tv3, &g1sswuCurveBCoeff) // 6.  tv3 = B * tv3
```

Comment:
iirc this is a coefficient of an isogenous curve, not our curve itself. That should be reflected in its name.

Reply:
Renamed